integration/mountkexec_test.go

```diff
@@ -0,0 +1,38 @@
+package integration
+
+import (
+	"testing"
+)
+
+// TestMountKexec runs an init which mounts a filesystem and kexecs a kernel.
+func TestKexecMount(t *testing.T) {
+	// TODO: support arm
+	if TestArch() != "amd64" {
+		t.Skipf("test not supported on %s", TestArch())
+	}
+
+	// Create the CPIO and start QEMU.
+	q, cleanup := QEMUTest(t, &Options{
+		Cmds: []string{
+			"github.com/u-root/webboot/integration/testcmd/mountkexec/uinit",
+			"github.com/u-root/u-root/cmds/core/init",
+			"github.com/u-root/u-root/cmds/core/ls",
+		},
+	})
+	defer cleanup()
+
+	if err := q.Expect("error making mount directory:mkdir : no such file or directory"); err != nil {
```

Comment:
since this is a fixed set of strings would it be possible to do a for loop over a p[]string? you have lots of repeated code here.

Reply:
Done.This is more efficient.